doto-frontend/src/components/ModalContent.js

```diff
@@ -85,11 +85,11 @@ const ModalContent = props => {
             reminder: selectedReminder,
         };
 
-        props.addTask(task, new Date());
+        props.addNewTask(task, new Date());
     };
 
     return (
-        <div className={props.modalBackground ? "modal-p" : "modal-g"}>
+        <div className={props.modalBackground === "dark" ? "modal-p" : "modal-g"}>
```

Comment:
"dark" and similar predefined strings might be more appropriately defined within an enum